providers/implementations/ciphers/cipher_cts.c

```diff
@@ -8,10 +8,10 @@
  */
 
 /*
- * Helper functions for AES CBC CTS ciphers.
+ * Helper functions for AES/CAMELLIA CBC CTS ciphers.
```

Comment:
Perhaps - `Helper functions for 128bit block size CBC CTS ciphers (currently AES/CAMELLIA)`